pandas/io/tests/test_excel.py

```diff
@@ -490,6 +490,13 @@ def test_date_conversion_overflow(self):
         result = self.get_exceldf('testdateoverflow')
         tm.assert_frame_equal(result, expected)
 
+    def test_true_and_false_value_options(self):
+        # GH 13347 : Allow true_values and false_values options in read_excel
+        expected = pd.DataFrame([[True, False]], columns=['col1', 'col2'])
+        result = self.get_exceldf('truefalsevalues', true_values=['foo'],
+                                  false_values=['bar'])
+        tm.assert_frame_equal(result, expected)
+
```

Comment:
instead of the included sheets I would prefer a round-trip example (e.g. write strings and float), e.g. multiple columns, then interpret them on the read-back.